3dhubs_amsterdam_python_flask.html

```diff
@@ -0,0 +1,64 @@
+---
+title:  Python Developer
+company: 3D Hubs
+url: https://www.3dhubs.com/
+location: Amsterdam, the Netherlands
+contract: temporary but can be converted to permanent 
+contact:
+    name: Mrs A. Cojanu
+    email: alina@3dhubs.com
+    phone: 0031 6 81 11 41 17
+created: !!timestamp '2017-01-27' 
+tags:
+  - the Netherlands
+  - Amsterdam
+  - python
+  - postgresql
+  - flask
+  - medior 
```

Comment:
Is this a spelling mistake or a foreign-language word? At the moment we don't have a policy against foreign language ads, however it may cause confusion if you are mixing.